conplex_dti/cli/predict.py

```diff
@@ -0,0 +1,108 @@
+from argparse import ArgumentParser
+
+import numpy as np
+import pandas as pd
+import torch
+from torch.utils.data import DataLoader
+
+from conplex_dti.featurizer import MorganFeaturizer, ProtBertFeaturizer
+from conplex_dti.model.architectures import SimpleCoembeddingNoSigmoid
+from conplex_dti.utils import set_random_seed
+
+
+def add_args(parser: ArgumentParser):
+    parser.add_argument(
+        "--data-file",
+        type=str,
+        required=True,
+        default="./data/pairs.tsv",
+        help="Path to the file containing data in the form [proteinID] [moleculeID] [proteinSequence] [moleculeSmiles].",
+    )
+    parser.add_argument(
+        "--model-path",
+        type=str,
+        required=False,
+        default="./models/ConPLex_v1_BindingDB.pt",
+        help="Path to the file containing the model to use for predictions. Default: ./models/ConPLex_v1_BindingDB.pt",
+    )
+    parser.add_argument(
+        "--results-ofile",
+        type=str,
+        required=False,
+        default="results.tsv",
+        help="Filename to save the resulting dataframe. Default: results.tsv",
+        # NOTE: the table is stored in the form [moleculeID] [proteinID] [prediction]
+    )
+    parser.add_argument(
+        "--features-dir",
+        type=str,
+        required=False,
+        default=".",
+        help="Directory to store the Morgan features and ProtBert .h5 files that are created within to the program. Default: .",
+    )
+    parser.add_argument(
+        "--random-seed",
+        type=int,
+        required=False,
+        default=61998,
+        help="Random seed to use for reproducibility. Default: 61998",
+    )
+    return parser
+
+
+def main(args):
+    set_random_seed(args.random_seed)
+
+    try:
+        query_df = pd.read_csv(
+            args.data_file,
+            sep="\t",
+            names=["proteinID", "moleculeID", "proteinSequence", "moleculeSmiles"],
+        )
+    except FileNotFoundError:
+        print(f"Could not find data file: {args.data_file}")
```

Comment:
Please replace print statements with `logg.info` -- see `cli/` train.py` for examples (including `from ..utils import config_logger`, etc.